dragonfly/session/session.go

```diff
@@ -161,6 +161,23 @@ func (s *Session) SendJukeBoxPopup(message string) {
 	})
 }
 
+// SendScoreboard ...
+func (s *Session) SendScoreboard(displayName string, objName string){
+	s.writePacket(&packet.SetDisplayObjective{
+		DisplaySlot:   "sidebar",
+		ObjectiveName: objName,
+		DisplayName:   displayName,
+		CriteriaName:  "dummy",
+		SortOrder:     0,
```

Comment:
You can remove this line due to the default value. Also, have you checked if the "dummy" criteria name is required?

Reply:
looking at other implementations - they used the dummy criteria name, hence why i added it there

Reply:
Yeah I understand. Have you tried to see if it works when keeping this field empty as well?

Reply:
Just tried it, removing the sort order does nothing and allows the scoreboard to be displayed, but having the criteria name to nothing (or removing it entirely) doesn't display the scoreboard at all

Reply:
Okay, thanks for checking 👍